.vscode-template/settings.json

```diff
@@ -0,0 +1,38 @@
+{
+  "editor.tabSize": 2,
+  "editor.insertSpaces": true,
+  "editor.detectIndentation": false,
+  "files.associations": {
+    ".*rc": "ini",
+    ".env*": "ini",
+    "Dockerfile*": "dockerfile",
+    "**/requirements/*.txt": "pip-requirements",
+    "**/requirements/*.in": "pip-requirements"
+  },
+  "files.eol": "\n",
+  "files.insertFinalNewline": true,
+  "files.trimFinalNewlines": true,
+  "files.trimTrailingWhitespace": true,
+  "python.formatting.autopep8Args": [
+    "--max-line-length 140"
+  ],
+  "[python]": {
+    "editor.detectIndentation": false,
+    "editor.tabSize": 4
+  },
+  "[makefile]": {
+    "editor.insertSpaces": false
+  },
+  "python.testing.pytestArgs": [
+    "services/web/server"
```

Comment:
this is probably not going to work for this repo

Reply:
i copied the wrong one... thx